services/afk_service.py

```
@@ -0,0 +1,81 @@
+        await execute_action(query, (user_id,))
+
+    async def is_afk(self, user_id: str) -> bool:
+        """Check if a user is currently AFK."""
+        query = "SELECT 1 FROM afk_users WHERE user_id = %s"
+        result = await safe_execute_query(query, (user_id,))
+        return result is not None and len(result) > 0
+
+    async def get_reason(self, user_id: str) -> str | None:
+        """Get the AFK reason for a user."""
+        query = "SELECT reason FROM afk_users WHERE user_id = %s"
+        result = await execute_query(query, (user_id,))
```

Flake8 (7.3.0)

[error] undefined name 'execute_query'

(F821)